app/mailers/user_mailer.rb

```diff
@@ -1,24 +1,28 @@
 class UserMailer < ApplicationMailer
+  def welcome_email(user, organization)
+    build_email user, :welcome, { inline: organization.welcome_email_template }, organization
```

Comment:
Could we make welcome use the organization display_name if available?
Something like Welcome to + (organization.display_name || "your new organization")! ?

Reply:
done!
Had to move the call to `I18n.with_locale` up so it would include the subject.
Also ended up using `organization[:display_name]` because I couldn't come up with a better name for the method that's already defined, any ideas? :thinking: